CommonTools/PileupAlgos/python/Puppi_cff.py

```diff
@@ -30,7 +30,8 @@
                        UseFromPVLooseTight = cms.bool(False),
                        UseDeltaZCut   = cms.bool(True),
                        DeltaZCut      = cms.double(0.3),
-		       PtMaxCharged   = cms.double(0.),
+		       PtMaxCharged   = cms.double(-1.),
+		       EtaMaxCharged   = cms.double(99999.),
 		       PtMaxPhotons = cms.double(0),
```

Comment:
For analogy, please also initialize to "-1" ```PtMaxPhotons```, both here and in the fillDescrption method